internal/interfaces/grpc/handler/operator.go

```diff
@@ -215,10 +215,10 @@ func (o operatorHandler) listDeposits(
 	deposits, err := o.operatorSvc.ListDeposits(
 		ctx,
 		int(req.GetAccountIndex()),
-		domain.Page{
-			Number: int(req.GetPage().GetPageNumber()),
-			Size:   int(req.GetPage().GetPageSize()),
-		},
+		domain.NewPage(
+			int(req.GetPage().GetPageNumber()),
```

Comment:
This is still not panic safe! If in the request no page is defined, here GetPage() will return nil and trying to access GetPageNumber from a nil pointer will make the daemon to panic. In case GetPage() returns nil we simply create a default page domain like domain.NewPage(0,0).

Reply:
@altafan now ok?

Reply:
Actually no: this way if the page and number in the request are invalid they are still passed down because their validation is only done in domain.NewPage.
The code here should be like this IMO:
```
page := domain.NewPage(0,0)
if req.GetPage() != nil {
   page = domain.NewPage(
       int(req.GetPage().GetPageNumber()),
       int(req.GetPage().GetPageSize()),
   )
}
```

Reply:
Done